components/match2/commons/match_group_util.lua

```diff
@@ -438,92 +481,36 @@ function MatchGroupUtil.gameFromRecord(record)
 	}
 end
 
-function MatchGroupUtil.computeUpperMatchIds(matchesById)
-	local upperMatchIds = {}
-	for matchId, match in pairs(matchesById) do
-		if match.bracketData.type == 'bracket' then
-			for _, lowerMatchId in ipairs(match.bracketData.lowerMatchIds) do
-				upperMatchIds[lowerMatchId] = matchId
-			end
-		end
+function MatchGroupUtil.computeLowerMatchIdsFromLegacy(data)
```

Comment:
Is this lower as in the right child of a node in a tree (same definition as LPDB) or lower as in any child of a node?

Reply:
It refers to ids of matches from lower rounds that advance to this match.